src/Evolutionary.jl

```diff
@@ -17,7 +17,6 @@ using Random
            es, cmaes, ga
 
     const Strategy = Dict{Symbol,Any}
-    const Individual = Union{Vector, Matrix, Function, Nothing}
```

Comment:
Why did you drop this? `Matrix` type can be used for fixed size population. Removing `Function` would be enough as you introduce new parameter for functional creation.

Reply:
I didn't find the 'Individual' type to be easy to understand - I don't think it represents an 'individual' at all, but rather the type of initialiser. As this PR breaks down initialisation into two parameters, one which must be a function, and one which must be a vector, I felt that the union type was no longer necessary.